pyfar/dsp/filter/gammatone.py

```diff
@@ -309,7 +309,7 @@ def _get_delays_and_phase_factors(self):
             delay_samples + 3, sampling_rate=self.sampling_rate))
 
         # compute the envelope
-        ir = real.time + 1j * imag.time
+        ir = real.time[:, 0, :] + 1j * imag.time[:, 0, :]
```

Comment:
```suggestion
        ir = real.time[..., 0, :] + 1j * imag.time[..., 0, :]
```
I am not sure about this, it looks very dangerous. Is there always a new dimension at dim=-2?

Reply:
It should be always at dim=-2.  The impulse being used in `_get_delays_and_phase_factors` only varies in length and sampling rate.

Reply:
Yes, I think it is fine this way